package.json

```diff
@@ -859,6 +886,7 @@
     "fs-extra": "^6.0.1",
     "glob": "7.1.2",
     "gradle-to-js": "^1.0.1",
+    "handlebars": "^4.0.11",
```

Comment:
I thought we agreed handlebars wasn't being used?

Reply:
we did. When I updated the branch some changes reversed. I thought I catch them all. Sorry.